src/app/public/modules/validators/validators.ts

```diff
@@ -19,7 +19,7 @@ export class SkyValidators {
    * errors or `undefined` immediately when users enter values.
    * @param control
    */
-  public static email(control: AbstractControl): ValidationErrors | null {
+  public static email(control: AbstractControl): ValidationErrors | undefined {
```

Comment:
Should we instead return `ValidationErrors | null` since the original API of the method is `ValidationErrors | null`?